baselines/exo.md

```diff
@@ -994,18 +1000,29 @@ The alerts SHOULD be sent to a monitored address or incorporated into a SIEM.
 
 ## 17. Microsoft Purview Audit
 
-Unified audit logging generates logs of user activity in M365 services.
-These logs are essential for conducting incident response and threat detection activity.
+User activity from M365 services are captured in the organization's unified
+audit log.  These logs are essential for conducting incident response and
+threat detection activity.
 
-By default, Microsoft retains the audit logs for only 90 days. Activity by users with E5 licenses is logged for one year.
+By default, Microsoft retains the audit logs for only 90 days. Activity by
```

Comment:
Please update this per the recent announcement to note that this will be extended to 180 days and will expand to include purview premium events as well.

https://www.microsoft.com/en-us/security/blog/2023/07/19/expanding-cloud-logging-to-give-customers-deeper-security-visibility/